imap_processing/idex/idex_l1b.py

```diff
@@ -0,0 +1,175 @@
+"""IMAP-IDEX L1B processing module."""
```

Comment:
```suggestion
"""IDEX L1B processing module."""
```

It would also be great if you could add an example here of how the module is run.